astarte/device/interface.py

```diff
@@ -81,10 +81,10 @@ def __init__(self, interface_definition: dict):
         self.type: str = interface_definition["type"]
         self.ownership = interface_definition.get("ownership", DEVICE)
         self.aggregation = interface_definition.get("aggregation", "")
-        self.mappings = {}
+        self.mappings = []
```

Comment:
Is this change safe for preventing duplicate mappings? Is it because there cannot be duplicates in the JSON de serialization?

Maybe a comment for the reasoning to make it cleared could be helpful?

Reply:
True, I added a check for it.